src/txmempool.cpp

```diff
@@ -282,24 +285,26 @@ void CTxMemPool::UpdateForRemoveFromMempool(const setEntries &entriesToRemove, b
         // should be a bit faster.
         // However, if we happen to be in the middle of processing a reorg, then
         // the mempool can be in an inconsistent state.  In this case, the set
-        // of ancestors reachable via mapLinks will be the same as the set of
-        // ancestors whose packages include this transaction, because when we
-        // add a new transaction to the mempool in addUnchecked(), we assume it
-        // has no children, and in the case of a reorg where that assumption is
-        // false, the in-mempool children aren't linked to the in-block tx's
-        // until UpdateTransactionsFromBlock() is called.
+        // of ancestors reachable via GetMemPoolParents()/GetMemPoolChildren()
```

Comment:
I'm not sure if it making sense to refer to `GetMemPoolChildren`. Again I understand this comment it's saying that during a reorg `mapTx` entries won't have their parent set correctly setup until `UpdateTransactionsFromBlock` thus relying on `mapLinks` to fetch the correct parent set, as this one is setup at `addUnchecked`.

Reply:
I'm not really sure what you're suggesting but I think the documentation can be cleaned up in the future.